zzk/service/hoststate_test.go

```diff
@@ -33,27 +33,23 @@ var (
 	ErrTestNoAttach = errors.New("could not attach to container")
 )
 
+const (
+	servicePath string = "/services/serviceid"
+	hostPath    string = "/hosts/hostid"
+	hostId      string = "hostid"
+	serviceId   string = "serviceid"
+	serviceName string = "serviceA"
+	imageId     string = "imageid"
+	containerId string = "containerid"
+)
+
```

Comment:
This is kind of a nitpick, but I don't like that you set these as constants, because there may be tests in the future that may require different values for these attributes